### ISSUE TITLE
Steam claims Half-Life: Alyx is running when it isn't

### ISSUE THREAD

**Issue by TheGubbinsCat** [ValveSoftware/steam-for-linux#7122]
#### Your system information

* Steam client version: 2020-05-21
* Distribution: Arch Linux (Zen kernel)
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
Since the Linux version released, Alyx has started shown as running on my profile without any corresponding processes running. The typical solution to this is to restart steam but that's entirely ineffective due to the game apparently opening itself whenever i open steam or steamvr.

My current workaround is to run the windows hlvr.exe as a non-steam game through proton which makes steam realise that it isn't really running, despite me currently having the native linux version installed.

####Not the steps for reproduction, sorry:
As far as I can tell, it more precisely started happening as soon I messed around with proton for the game, along with my attempts to get the workshop tools working. At the same time, my steam friends list started failing to show up which seems to be related.

**Comment by kisak-valve** [comment]
Possibly related to #6220.

**Comment by TheGubbinsCat** [comment]
I've now fixed the issues I was experiencing. I renamed steam_appid.txt and my userdata localconfig.vdf so they wouldn't be picked up. I can't verify if steam_appid.txt was actually affecting anything however it contained the appid for Alyx which is why I think it could be related. Resetting localconfig.vdf got my friends list working again and after getting back to steam it wasn't automatically claiming Alyx was running anymore.
